### DISCUSSION TITLE
Search options (not only docs, but also blogs and guides)

### DISCUSSION THREAD

**Discussion by stephanbruijnis** [thuliteio/doks#778] [Q&A] [answered]
Hi,

I'm considering to switch my site to Doks theme for Hugo. But I have some questions about the search options

1) I noticed that search only seems to return results from the Docs section and not any other (FAQ, Guide, Blogs). Would it be possible to extend this to all sections?

With kind regards,
Stephan

**Comment by h-enk** [comment, marked as answer]
Yes. It will be part of the next release — see also: https://github.com/h-enk/doks/issues/630

Until then, see https://github.com/h-enk/doks/discussions/395
